tms_objects2.erb

```diff
@@ -266,10 +290,19 @@ FileUtils.mkdir('records')
 
 #ITERATING THROUGH OBJECTS
 #objects_main_query = tmsclient.execute("SELECT top 10 * FROM [Coboat_ObjectsMainQuery]").each
-objects_main_query = tmsclient.execute("SELECT * FROM [Coboat_ObjectsMainQuery] where ObjectID in (22,6236,515)").each
+#objects_main_query = tmsclient.execute("SELECT * FROM [Coboat_ObjectsMainQuery] where ObjectID in (22,6236,515)").each
+objects_main_query = tmsclient.execute("SELECT * FROM [Coboat_ObjectsMainQuery] where ObjectID in (61579,755,1491)").each
 #22 - main test object
-#6236 - for displayStateEditionWrap
-#515 - for qualifierMeasurements
+#6236 - Production displayStateEditionWrap - State - trial touched pr
```

Comment:
the production event only applies to the nameActorset changes event, remove from 4 lines displayStateEditions and qualifier measuremnts